tests/two_sum_test.py

```diff
@@ -1,2 +1,37 @@
 import pytest
 from functions.two_sum import twoSum
+
+def test_returns_0_1_for_2_7_11_15_with_target_9():
+    # Arrange
+    n = [2, 7, 11, 15]
+    t = 9
+
+    # Act
+    answer = twoSum(n, t)
+
+    # Assert
+    assert answer == [0, 1]
+
+
+def test_returns_1_2_for_3_2_4_with_target_6():
+    # Arrange
+    n = [3, 2, 4]
+    t = 6
+
+    # Act
+    answer = twoSum(n, t)
+
+    # Assert
+    assert answer == [1, 2]
+
+
+def test_returns_0_1_for_3_3_with_target_6():
```

Comment:
these tests look great-they have really clear arrange, act, and assert steps. we could probably add some edge case tests too!